src/components/UI/Dropdown/types.ts

```diff
@@ -1,6 +1,6 @@
 export type Options = {
```

Comment:
почему во множественном числе, когда это тип **одной** опции
```suggestion
export type Option = {
```